hints/daily-temperatures.md

```diff
@@ -0,0 +1,39 @@
+<br>
+<details class="hint-accordion">  
+    <summary>Recommended Time & Space Complexity</summary>
+    <p>
+    You should aim for a solution as good or better than <code>O(n)</code> time and <code>O(n)</code> space, where <code>n</code> is the size of the input array.
+    </p>
+</details>
+
+<br>
+<details class="hint-accordion">  
+    <summary>Hint 1</summary>
+    <p>
+    A brute force solution would involve iterating through the array with index <code>i</code> and checking how far is the next greater element to the right of <code>i</code>. This would be an <code>O(n^2)</code> solution. Can you think of a better way?
+    </p>
+</details>
+
+<br>
+<details class="hint-accordion">  
+    <summary>Hint 2</summary>
+    <p>
+    Can you consider a reverse approach? For example, in <code>[2, 1, 1, 3]</code>, the next greater element for the numbers <code>[2, 1, 1]</code> is <code>3</code>. Instead of checking for each element individually, can you think of a way where, by standing at the element <code>3</code>, you compute the result for the elements <code>[2, 1, 1]</code>?  Maybe there's a data structure that is useful here.
+    </p>
+</details>
+
+<br>
+<details class="hint-accordion">  
+    <summary>Hint 3</summary>
+    <p>
+   We can use a stack to maintain elements in a monotonically decreasing order. To maintain the monotonically decreasing order, we pop elements from the stack whose values are smaller than the current element. However, since we need to compute the result array by finding the difference between indices, we should store the indices of the elements in the stack instead of the elements themselves but should consider the values at that indices while operating the stack. Can you figure out how is this stack useful?
```

Comment:
> However, since we need to compute the result array by finding the difference between indices, we should store the indices of the elements in the stack instead of the elements themselves but should consider the values at that indices while operating the stack.

This sentence is pretty long, can we split it up.